packages/browser-destinations/src/destinations/intercom/index.ts

```diff
@@ -0,0 +1,71 @@
+import type { Settings } from './generated-types'
+import type { BrowserDestinationDefinition } from '../../lib/browser-destinations'
+import { browserDestination } from '../../runtime/shim'
+import { initialBoot, initScript } from './init-script'
+
+import trackEvent from './trackEvent'
+import { Intercom } from './api'
+
+import updateUser from './updateUser'
+
+import updateCompany from './updateCompany'
```

Comment:
Nit on order and spacing:

```suggestion
import { Intercom } from './api'
import trackEvent from './trackEvent'
import updateUser from './updateUser'
import updateCompany from './updateCompany'
```